admin-portal/src/components/navbar/navbar.tsx

```diff
@@ -18,9 +18,9 @@ export function NavBar(params: NavBarProps) {
             <svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true">
               <path
                 stroke="currentColor"
-                stroke-linecap="round"
-                stroke-miterlimit="10"
-                stroke-width="2"
+                strokeLinecap="round"
+                strokeMiterlimit="10"
+                strokeWidth="2"
```

Comment:
Nice! The warnings were bugging me :D

Reply:
Haha me too 😁

Reply:
Thanks for the fix, I'll mark this as resolved.